aiowebostv/webos_client.py

```diff
@@ -468,10 +469,10 @@ async def set_power_state(self, payload):
             await self.do_state_update_callbacks()
 
     async def set_current_app_state(self, app_id):
-        """Set current app state variable.
-        This function also handles subscriptions to current channel and channel list,
-        since the current channel subscription can only succeed when Live TV is running,
-        and the channel list subscription can only succeed after channels have been configured."""
+        """Set current app state variable."""
+        # This function also handles subscriptions to current channel and channel list,
```

Comment:
A docstring is either a single line within 72 characters or a header line followed by a blank line and then a body with possibly multiple lines. No line should be longer than 72 characters.

We can include the comments here in the docstring body.